Makefile

```diff
@@ -1,23 +1,25 @@
-DOCKER_REPO ?= gcr.io/online-bridge-hackathon-2020
+# Project / Org
+GCP_PROJECT ?= globalbridge-app
+GKE_ZONE ?= europe-west4-b
+
+# Service / App
+GKE_CLUSTER_NAME ?= prod-cluster
+RELEASE_NAME ?= dds
+K8S_NS ?= prod-${RELEASE_NAME}
```

Comment:
Should default be testing servers?

Then release parameters could be selected using `make push DEPLOY_TARGET=release`.

Release deploy might even require some extra steps to ensure upgrade is as smooth as possible. I think well designed upgrade process could even do the upgrade while letting user continue play. But this is again future problems instead of immediate issues.